voluncheer/profiles/test_views.py

```diff
@@ -12,7 +12,7 @@ def test_signup_form_accessible_by_name(self):
 
     def test_signup_form_exists_at_desired_location(self):
         """Tests accessing  signup form by URL"""
-        response = self.client.get("/accounts/signup/")
```

Comment:
I think unless you change LOGIN_REDIRECT_URL in settings.py, we should have have URL starting with accounts:
https://stackoverflow.com/questions/13186693/why-second-user-login-redirects-me-to-accounts-profile-url
Is the URL accessible after the change?